### ISSUE TITLE
[BUG] Not working on Debian 10

### ISSUE THREAD

**Issue by Lucif3r-MorningSter** [toulousain79/MySB#100]
After fresh install on Debian 10 server does not start after reboot and seedbox not reachable.

**Comment by toulousain79** [comment]
Hi
Maybe you don't respect prerequisites...
Please check it on https://mysb.gitbook.io/doc/installation/pre-requis

And come back with a confirmation

Bye

**Comment by Lucif3r-MorningSter** [comment]
All prerequisites has present. I have installed it on debian 9 before.

**Comment by toulousain79** [comment]
So, you come from v6.0, and it's the upgrade that failed.

Please, come to Discord, and send me an archive of /opt/MySB/logs/ content
in private message.
I will investigate logs files.
Afterwards, I will maybe take hand with AnyDesk for a debug.

**Comment by toulousain79** [comment]
Hi
I'm very busy this time.
So, I'll can help you but you must be patient...

Where is located your server, which provider?

**Comment by Lucif3r-MorningSter** [comment]
My server is from hetzner. what is discord address?

**Comment by toulousain79** [comment]
https://github.com/toulousain79/MySB/wiki/%5BHelp%5D-Support

**Comment by toulousain79** [comment]
Hi
I have found the problem with Hetzner.
The Hetzner team had a new config file with Buster to force IPv6: /etc/systemd/network/10-mainif.network
I must find solution for comment or remove this 2 lines in it.
```
[Network]
Address=2a01:4f8:172:342::2/64
Gateway=fe80::1
```

After renting a new server, I have checked this with deleting this lines and solution works !
I must find the lines code  to do this automaticly.
